draft-terbu-sd-jwt-vc.md

```diff
@@ -255,6 +255,12 @@ exist between `sub` and `cnf` claims.
 Additional public claims MAY be used in SD-JWT VCs depending on the
 application.
 
+#### SD-JWT VC without selectively disclosable claims
+
+SD-JWT VC may not contain any selectively disclosable claims.
+Such SD-JWT VC does not contain `_sd` claim in the JWT body and
+does not have any Disclosures.
```

Comment:
```suggestion
In that case, the SD-JWT VC MUST NOT contain the `_sd` claim in the JWT body and
MUST NOT have any Disclosures.
```

Reply:
```suggestion
In that case, the SD-JWT VC MUST NOT contain the `_sd` claim in the JWT body. I also
MUST NOT have any Disclosures.
```

Reply:
I prefer avoiding two normative requirements (MUST NOTs) in the same sentence.